compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs

```diff
@@ -810,15 +808,36 @@ renderValue world name = \case
         renderField (Field label mbValue) =
             renderValue world (name ++ [TL.toStrict label]) (fromJust mbValue)
 
+templateConName :: LF.World -> Identifier -> Maybe (LF.Qualified LF.TypeConName)
```

Comment:
Is there no helper function in the LF world that does this lookup for us? If not, should there be?